core/controllers/reader.py

```diff
@@ -1005,15 +1005,25 @@ def get(self):
 
         skill_ids = self.request.get('skill_ids').split(',')
         question_count = self.request.get('question_count')
+        fetch_by_mastery = self.request.get('fetch_by_mastery')
```

Comment:
Change variable name to fetch_by_mastery_value

Reply:
Done.